### PR TITLE
Update FAQ to indicate legacy opcodes were removed

### PR DESCRIPTION
**Pull request by alexbevi**
Also remove mention of upsert in the FAQ about client-side _id generation

Please complete the following before merging:

- [x] Update changelog.
- [ ] Make sure there are generated JSON files from the YAML test files.
- [ ] Test changes in at least one language driver.
- [ ] Test these changes against all server versions and topologies (including standalone, replica set, sharded
  clusters, and serverless).

### CODE REVIEW

File: source/server_write_commands.rst [hunk 1]
```diff
@@ -477,7 +477,9 @@ Prior to MongoDB 3.6, an ``insert`` operation would use the  ``OP_INSERT`` opcod
 Can a driver still use the ``OP_INSERT``, ``OP_DELETE``, ``OP_UPDATE``?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Yes, a 2.6 server will still support those. But it is unlikely that a 2.8 server would.  Of course, when talking to older servers, the usual op codes will continue working the same. An older server is one that reports ``hello.maxWireVersion`` to be less than 2 or does not include the field.
+The `legacy opcodes were removed in MongoDB 6.0 <https://www.mongodb.com/docs/manual/release-notes/6.0-compatibility/#legacy-opcodes-removed>`_.
+
+For historical context, a 2.6 server would still support those, however it is unlikely that a 2.8 server would.  Of course, when talking to older servers, the usual op codes will continue working the same. An older server is one that reports ``hello.maxWireVersion`` to be less than 2 or does not include the field.
```
Comment (alcaeus) [inline source/server_write_commands.rst:482]: Wait, if they're only removed in 6.0, why is it unlikely that a 2.8 server still would support them? 🤔

Comment (alexbevi) [reply source/server_write_commands.rst:482]: Lol, that was the original text. I guess if I'm modifying this anyway there's no harm in removing the old content

Comment (alexbevi) [reply source/server_write_commands.rst]: I've updated the text to clarify

File: source/server_write_commands.rst [hunk 2]
```diff
@@ -477,12 +477,7 @@ Prior to MongoDB 3.6, an ``insert`` operation would use the  ``OP_INSERT`` opcod
 Can a driver still use the ``OP_INSERT``, ``OP_DELETE``, ``OP_UPDATE``?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The `legacy opcodes were removed in MongoDB 6.0 <https://www.mongodb.com/docs/manual/release-notes/6.0-compatibility/#legacy-opcodes-removed>`_.
-
-For historical context, a 2.6 server would still support those, however it is unlikely that a 2.8 server would.  Of course, when talking to older servers, the usual op codes will continue working the same. An older server is one that reports ``hello.maxWireVersion`` to be less than 2 or does not include the field.
-
-The rationale here is that we may choose to divert all the write traffic to the new
-protocol. (This depends on the having the overhead to issue a batch with one item very low.)
+The `legacy opcodes were removed in MongoDB 6.0 <https://www.mongodb.com/docs/manual/release-notes/6.0-compatibility/#legacy-opcodes-removed>`_. As of MongoDB 3.6 these opcodes were superceded by `OP_MSG <https://www.mongodb.com/docs/manual/reference/mongodb-wire-protocol/#op_msg>`_, however all server versions up until 6.0 continued to support the legacy opcodes.
```
Comment (alcaeus) [inline source/server_write_commands.rst:480]:
```suggestion
The `legacy opcodes were removed in MongoDB 6.0 <https://www.mongodb.com/docs/manual/release-notes/6.0-compatibility/#legacy-opcodes-removed>`_. As of MongoDB 3.6 these opcodes were superseded by `OP_MSG <https://www.mongodb.com/docs/manual/reference/mongodb-wire-protocol/#op_msg>`_, however all server versions up until 6.0 continued to support the legacy opcodes.
```

**Review by alcaeus — APPROVED** [review]
LGTM with the typo fixed 😉

**Pull request merged**